docs/spatial_dbms.rst

```diff
@@ -6,7 +6,7 @@ Supporting installation of spatial data into `Postgres DBMS`.
 
 1. **Install Postgres**
 
-  To install `Postgres` from scratch please ref to `Postgres installation`_.
+  To install `Postgres` from scratch please ref to `Postgres installation`_ or use a much simple installation method with `Postgres.app`_.
```

Comment:
Postgres.app is just for macos and it looks like the old installation link was just for Linux. Should we include a recommended link for each OS for installation?